projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/domain/query-management.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Document } from '../../../../domain/document/document-data.model';
+
+export interface PartyMessage {
+  id: string;
+  subject?: string;
+  body: string;
+  attachments?: Document[];
+  isHearingRelated: boolean;
+  hearingDate?: Date;
+  createdOn: Date;
+  createdBy: string;
+  parentId?: string;
```

Comment:
missing name field

Reply:
I have added the name field and also updated the mock data.